### ISSUE TITLE
Support later Java & jfxrt

### ISSUE THREAD

**Issue by hodgett** [sleuthkit/autopsy#5817]
Updates to java8 have stopped. Updates to jfx in 8 have stopped. Please bring autopsy up to date already with it's java support, it would help resolve so many peoples (installation)  problems!

**Comment by jonas-koeritz** [comment]
Moving to a newer NetBeans version would also help. I don't have much experience using NetBeans but switching to the current platform would be good.

**Comment by rcordovano** [comment]
Basis Technology plans to look at updating the versions of Java, JavaFX, and NetBeans that Autopsy depends on for the version 4.16.0 release tentatively scheduled for the late June, early July time frame. Note that Autopsy 4.15.0 will be released next week.

**Comment by jonas-koeritz** [comment]
That's great news! Thanks for keeping us updated.

**Comment by noraj** [comment]
OpenJDK 8 is still in ArchLinux repository but OpenJFX 8 has been dropped.

Oracle Java SE 8 Premier Support Until as ended in March 2022 (https://www.oracle.com/java/technologies/java-se-support-roadmap.html).

On Debian only stretch/9 still use OpenJFX 8, buster/10, bullseye/11, bookworm/12 use OpenJFX 11.
Same for Ubuntu only 16.04 use OpenJFX 8, 18.04, 20.04, 22.04, 22.10 use OpenJSX 11.
Pentoo use OpenJFX 17, Kali OpenJFX 11, etc.

In short nobody use OpenJFX 8 anymore outside of very old distro.

**Comment by AkechiShiro** [comment]
@rcordovano would you have any news on the updating of Java ? Or any pointers to where I can find some news update on who is now working on updating the java version of Autopsy ? Or has this been abandoned ?
